draft-ietf-ohai-ohttp.md

```diff
@@ -1134,6 +1134,24 @@ modified by a network attacker.  Note that a request could be forwarded without
 protection if the two resources share an origin.
 
 
+## Separate Gateway and Target
+
+A Target Resource that is operated on a different server from the Oblivious
+Gateway Resource is an ordinary HTTP resource.  A Target Resource can privilege
+requests that are forwarded by a given Oblivious Gateway Resource if it trusts
+the operator of the Oblivious Gateway Resource to only forward requests that
+meet the expectations of the Target Resource.
```

Comment:
```suggestion
meet the expectations of the Target Resource.  Otherwise, the Target Resource
treats requests from an Oblivious Gateway Resource no differently than any
other HTTP client.
```